### ISSUE TITLE
New rule to forbid to use `expect(find*(...)).toBeTruthy();`

### ISSUE THREAD

**Issue by retyui** [jest-community/eslint-plugin-jest#1423]
False Positives check, that need to be forbidden:

```
find*(...) // Promise<T>

expect(promise).toBeDefined(); // true
expect(promise).toBeTruthy(); // true
```

**Comment by SimenB** [comment]
We don't know the return type of your function, so I don't think there's anything we can do here?

**Comment by retyui** [comment]
sorry miss prj, should be RNTL eslint issue
